### ISSUE TITLE
Opt-out request for ThomCote

### ISSUE THREAD

**Issue by ThomCote** [bigcode-project/opt-out-v2#141]
I request that the following data is removed from The Stack and StackOverflow:

 - ThomCote/ColdCalling
 - ThomCote/GAM392
 - ThomCote/prime-colors
 - ALL

**Comment by ThomCote** [comment]
I'm notifying the "Software Heritage Foundation" of my demand that you take ALL code from my GitHub namespace ThomCote out of all versions of your AI model dataset, as well as your archive. You have violated the licenses of each repository you've scraped. The code must be removed entirely from your dataset.

The takedown procedure info as defined at https://www.softwareheritage.org/legal/content-policy/

1. Date: March 14, 2024
2. Identification: Thomas Cote, programmer. I am not legally required to provide my address, nationality, birth place or birth date.
3. Infringing information: All source code from GitHub repositories belonging to user ThomCote. This includes the following repositories as reported by https://huggingface.co/spaces/bigcode/in-the-stack:
ThomCote/ColdCalling
ThomCote/GAM392
ThomCote/prime-colors
4. Reason the data must be taken down: The source code was added to "The Stack" without my knowledge or permission. I added an explicit LICENSE file to each repository today, but prior to that my code had no license specified. Thus it was NOT public domain and NOT fair use. Additionally, the ColdCalling repo is a private GitHub repo and thus does not fall under the stated goal of "collect[ing] publicly available source code" written in the above content policy.
5. I am the author of the content whose license the Software Heritage Foundation has infringed upon.

Remove all of my code from your datasets and archives, and do not add any of my code in the future.

Thom Cote
